api/src/main/java/org/openmrs/Person.java

```diff
@@ -1100,5 +1100,63 @@ public void setId(Integer id) {
 		setPersonId(id);
 		
 	}
-	
+
+	/**
+	 * Convenience method to calculate this person's age in months based on the birthdate.
+	 *
+	 * @return Returns age in months as an Integer.
+	 * <strong>Should</strong> get correct age in months after death
+	 */
+	public Integer getAgeInMonths() {
```

Comment:
I'm not certain about adding methods like this; we very likely will need to support basically arbitrary-precision dates to handle neonates (where guidelines can be determined by hours, days, and weeks).

Reply:
Basically, rather than repeatedly extending `Person` it might be better to add a function to the Velocity context that can take a date and a `ChronoUnit` indicating the required precision.